app/services/subscriptions/dates_service.rb

```diff
@@ -151,7 +151,7 @@ def timezone_has_changed?
     end
 
     def previous_charge_to_datetime
-      return if last_invoice_subscription.blank?
+      return nil if last_invoice_subscription.blank?
```

Comment:
This change is not necessary, we can keep the original which does the same thing:

`return if last_invoice_subscription.blank?`